Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -1028,6 +1028,10 @@ theorem piLp_equiv_toEuclideanLin (A : Matrix m n 𝕜) (x : EuclideanSpace 𝕜
   rfl
 #align matrix.pi_Lp_equiv_to_euclidean_lin Matrix.piLp_equiv_toEuclideanLin
 
+theorem toEuclideanLin_apply (M : Matrix m n 𝕜) (v : EuclideanSpace 𝕜 n) :
+    toEuclideanLin M v =
+    (WithLp.equiv 2 (m → 𝕜)).symm (M *ᵥ (WithLp.equiv 2 (n → 𝕜)) v) := rfl
```

Comment:
style nit. The `rfl` on it's own line I generally do unless the entire statement fits on a single line. And when an expression takes more than one line, we generally indent the subsequent line an extra two spaces.
```suggestion
    toEuclideanLin M v =
      (WithLp.equiv 2 (m → 𝕜)).symm (M *ᵥ (WithLp.equiv 2 (n → 𝕜)) v) :=
  rfl
```